src/app/core/services/status.service.ts

```
@@ -45,194 +46,196 @@ export class StatusService {
     switch (true) {
       case lowerCaseComment.indexOf('automatically merged') > -1:
         statusCategory = {
-          category: 'Expense automatically merged',
+          category: this.translocoService.translate('services.status.expenseAutomaticallyMerged'),
           icon: 'merge',
         };
         break;
       case /(merged (\d+) expenses)/.test(lowerCaseComment):
         const regexMatch = lowerCaseComment.match(/merged (\d+) expenses/);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)